.circleci/config.yml~

```diff
@@ -0,0 +1,71 @@
+version: 2
```

Comment:
@snazzer Why are there 2, very similar config files? Is one just the original unmodified?

Reply:
This looks like vim backup file :)

Reply:
Fixed - thx.